containers/podman-desktop
# feat: update to new set of icons in navbar

### What does this PR do?
Update icons

### Screenshot/screencast of this PR

![image](https://user-images.githubusercontent.com/436777/194572915-4ecaae66-40d5-40a6-a9ec-678183d8112b.png)


### What issues does this PR fix or reference?

N/A

### How to test this PR?

look at what you see :-)

Change-Id: I6dc6af1b60fc761922e95001a464b3aedcfc5237
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
